### PR TITLE
Add Test Card

### PR DESCRIPTION
**Pull request by TheThing1000**
Add Test Card to demonstrate code reviews

### CODE REVIEW

File: src/components/Body.js [hunk 1]
```diff
@@ -4,10 +4,10 @@ const Body = () => {
   return (
     <main style={styles.body}>
       <div style={styles.card}>
-        <img src={"/avatar-default.png"} alt="John Doe" style={styles.photo} />
+        <img src={"/avatar-default.png"} alt="TEST" style={styles.photo} />
         <div style={styles.textContainer}>
-          <h2 style={styles.name}>John Doe</h2>
-          <p style={styles.position}>Software engineer</p>
```
Comment (AnastasiiaAdamovych) [inline src/components/Body.js:10]: Resolve conflicts. Put this code to the end.

**Review by AnastasiiaAdamovych — APPROVED** [review]
All good. Resolve conflicts (move code to the end of s container)

**Review by AnastasiiaAdamovych — APPROVED** [review]
All good. Ready to merge

**Review by Rostyk-01tm — APPROVED** [review]
Everything is good.

**Review by Ihnore-Ihor — APPROVED** [review]
All is good, merging

**Review by St01Yura — APPROVED** [review]
All good

**Pull request merged**